tests/IntegrationTest/Issues/Issue881Test.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DI\Test\IntegrationTest\Issues;
+
+use DI\ContainerBuilder;
+use DI\Test\IntegrationTest\BaseContainerTest;
+use DI\Test\IntegrationTest\Issues\Issue881\A;
+use DI\Test\IntegrationTest\Issues\Issue881\B;
+use DI\Test\IntegrationTest\Issues\Issue881\C;
+use DI\Test\IntegrationTest\Issues\Issue881\ClassWithVariadicParameter;
+use function DI\create;
+use function DI\get;
+
+final class Issue881Test extends BaseContainerTest
+{
+    public function testContainerWithVariadicParameters(): void
```

Comment:
Can you move this to `DI\Test\IntegrationTest\Definitions\CreateDefinitionTest` and change the signature like the other test methods to merge both tests in 1:

```suggestion
    /**
     * @dataProvider provideContainer
     */
    public function test_with_variadics(ContainerBuilder $builder)
```

Reply:
Done [8854fe6](https://github.com/PHP-DI/PHP-DI/pull/883/commits/8854fe64da554a36d93b3a79cdf5045745fce58b).